### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tberry2112**
1. Updated link on - 'Box Windows SDK Video Tutorial' to go to original source at: https://youtu.be/hqko0hxbaXU
2. Removed the 's' from developers.box.com, the site is at developer.box.com (no 's')

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @tberry2112, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by tberry2112** [comment]
CLA signed.

**Comment by boxcla** [comment]
Verified that @tberry2112 has just signed the CLA. Thanks, and we look forward to your contribution.

File: README.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ Other Resources
 -------------
 - Core SDK Nuget Package: https://www.nuget.org/packages/Box.V2/
 - JWT Support Nuget Package: https://www.nuget.org/packages/Box.V2.JWTAuth/
-- Box Windows SDK Video Tutorial: https://developers.box.com/box-windows-sdk-tutorial/
```
Comment (tberry2112) [inline README.md]: This link redirects to docs.box.com but the tutorial is not hosted on that site. Change this link to point to the YouTube version to resolve the broken link.

**Pull request merged**